### PR TITLE
[MS] Added list invitations, invite, and cancel invite using bindings

### PR DESCRIPTION
**Pull request by Max-7**
- [X] Keep changes in the pull request as small as possible
   
- [X] Ensure the commit history is sanitized
   
- [X] Give a meaningful title to your PR
- [X] Describe your changes

### CODE REVIEW

File: client/src/locales/fr-FR.json [hunk 1]
```diff
@@ -452,6 +452,13 @@
             "email": "Email",
             "date": "Invitation date",
             "status": "Status"
+        },
+        "CreateUserInvitationModal": {
+            "pageTitle": "Inviter un nouvel utilisateur",
+            "label": "E-mail de l'utilisateur",
+            "placeholder": "jean.martin{'@'}exemple.com",
```
Comment (FirelightFlagboy) [inline client/src/locales/fr-FR.json:459]:
```suggestion
            "placeholder": "jean.dujardin{'@'}exemple.com",
```

:thinking:

File: client/src/components/users/InvitationListItem.vue [hunk 2]
```diff
@@ -20,7 +20,8 @@
       <ion-label
         class="cell invitation-label"
       >
-        <span>{{ timeSince(invitation.date, '--', 'short') }}</span>
+        <span>{{ invitation.createdOn }}</span>
+        <!-- <span>{{ timeSince(invitation.createdOn, '--', 'short') }}</span> -->
```
Comment (NicoTuxx) [inline client/src/components/users/InvitationListItem.vue:24]: why do we keep this as comment?

File: client/src/views/users/ActiveUsersPage.vue [hunk 3]
```diff
@@ -170,6 +171,9 @@ import { UserAction } from '@/views/users/UserContextMenu.vue';
 import MsActionBar from '@/components/core/ms-action-bar/MsActionBar.vue';
 import { MockUser, getMockUsers } from '@/common/mocks';
 import { onMounted } from '@vue/runtime-core';
+import CreateUserInvitationModal from '@/views/users/CreateUserInvitationModal.vue';
+import * as Parsec from '@/common/parsec';
+import router, { routerNavigateTo } from '@/router';
```
Comment (NicoTuxx) [inline client/src/views/users/ActiveUsersPage.vue:176]: router seems not used

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: client/src/components/users/InvitationListItem.vue (hunk 2 above)
Comment (Max-7) [reply client/src/components/users/InvitationListItem.vue]: Fixed. It was because `createdOn` is a string, not a DateTime, but I added deserialization.

**Review by NicoTuxx — APPROVED** [review]

**Pull request merged**